.circleci/config.yml

```
@@ -120,7 +120,7 @@ jobs:
           name: Run tests
           command: |
             source venv/bin/activate
-            pytest --cov=networkx --cov-report=html -n 4 --runslow --doctest-modules --durations=20 --pyargs networkx
+            pytest --cov=networkx --cov-report=html -n 4 --doctest-modules --durations=20 --pyargs networkx
```

Somewhat unrelated to your PR, but are the doctests really doing anything useful in the `coverage` job?
I 100% think we should be testing the docstrings to make sure they don't become outdated (and we do in many of the other jobs), but if a doctest is the only place part of the code is covered, I'd prefer if that gets flagged for additional testing.
```suggestion
            pytest --cov=networkx --cov-report=html -n 4 --durations=20 --pyargs networkx
```

---

It's a good point and one that I was thinking myself. I'd vote to double-check that it there isn't a huge impact on coverage. I vote for checking in a follow-up PR so we can also see the difference in the coverage CI.

---

Sounds reasonable!